src/Tribe/Attendee_Repository.php

```diff
@@ -88,6 +106,21 @@ public function __construct() {
 			'rsvp_status'           => [ $this, 'filter_by_rsvp_status' ],
 		] );
 
+		// Add object default aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
+			'ticket_id'      => '_tribe_tickets_ticket_id',
+			'event_id'       => '_tribe_tickets_post_id',
+			'post_id'        => '_tribe_tickets_post_id',
+			'security_code'  => '_tribe_tickets_security_code',
+			'order_id'       => '_tribe_tickets_order_id',
+			'optout'         => '_tribe_tickets_optout',
+			'user_id'        => '_tribe_tickets_user_id',
+			'price_paid'     => '_tribe_tickets_price_paid',
+			'price_currency' => '_tribe_tickets_price_currency_symbol',
+			'full_name'      => '_tribe_tickets_full_name',
+			'email'          => '_tribe_tickets_email',
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 582eba347cc3f0933f779190dce5ddb2</sub>